### ISSUE TITLE
warning: you're about to play a dumb move

### ISSUE THREAD

**Issue by simondorfman** [skotz/volcanoes#33]
**When** a user plays on an empty space that will be destroyed on the next growth phase by a neighboring opponent's volcano.

**Or**

**When** a user plays a plus move on their own volcano that will be destroyed on the next growth phase by a neighboring opponent's volcano (and the plus move will not erupt now nor on the next growth phase).

**Then** display this Pop-up message:
"Are you sure? 12B will probably be destroyed on the next growth phase by a neighboring volcano eruption." 
Button: "Don't play here after all." [default]
Button: "Play here anyway."

**Comment by skotz** [comment]
Maybe there's some hidden strategy where playing on a tile that'll be instantly killed is the legitimate best move since you're blocking your opponent.

**Comment by simondorfman** [comment]
yeah, that's a good point. maybe the warning should only happen for your first move, before the growth between your own turns. If something you play gets destroyed on your own growth phase, then i'm pretty sure it's a mistake and you'd appreciate a warning to prompt you to take a second look.
